server/src/controllers/getBookById.js

```diff
@@ -0,0 +1,10 @@
+class getBook {
+  static async getBookById(req, res) {
+    return res.status(200).json({
+      status: 200,
+      data: [req.data],
```

Comment:
I don't understand this part, what's is data: [req.data] for

Reply:
I am attempting not to go into the database again so after I have checked the database for that book in the middleware I pass the book data into the req object as req.data so that I can send it as response.

Reply:
I hope you get this?